code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -340,6 +340,31 @@
 	tastes = list("muffin" = 3, "spookiness" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR | BREAKFAST
 
+/obj/item/reagent_containers/food/snacks/muffin/moffin
+	name = "moffin"
+	icon_state = "moffin"
+	desc = "A delicous and spongy little cake."
+	tastes = list("muffin" = 3, "dust" = 1, "lint" = 1)
+	foodtype = CLOTH | GRAIN | SUGAR | BREAKFAST
+
+/obj/item/reagent_containers/food/snacks/muffin/moffin/Initialize(mapload)
+	. = ..()
+	icon_state = "[icon_state]_[rand(1, 3)]"
+
+/obj/item/reagent_containers/food/snacks/muffin/moffin/examine(mob/user)
+	if(!ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/obj/item/organ/tongue/T = H.getorganslot(ORGAN_SLOT_TONGUE)
+
+		if(!T)
+			return
+
+		if(!HAS_TRAIT(H, TRAIT_AGEUSIA))
```

Comment:
Aguesia is taste based, why would ageusia affect your ability to look at and discern what is on the food?

Reply:
i havent fixed that examine yet, its cannibalized code atm